eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -11,7 +12,7 @@ stages:
       RunUnitTests: ${{parameters.RunUnitTests}}
 
 # The Prerelease and Release stages are conditioned on whether we are building a pull request and the branch.
-- ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+- ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'), eq(parameters.IncludeRelease,true))}}:
```

Comment:
@mitchdenny would know for sure but I think you need to wrap your true in quotes `'true'`

Reply:
```true``` should work.